operator/v1/types_ingress.go

```diff
@@ -716,8 +720,48 @@ type AWSNetworkLoadBalancerParameters struct {
 	// +optional
 	// +openshift:enable:FeatureGate=IngressControllerLBSubnetsAWS
 	Subnets *AWSSubnets `json:"subnets,omitempty"`
+
+	// eipAllocations is a list of IDs for Elastic IP (EIP) addresses that
+	// are assigned to the Network Load Balancer.
+	// The following restrictions apply:
+	//
+	// eipAllocations cannot be used with "Internal" but "External" scope.
+	// An EIP can be allocated to only a single IngressController.
+	// The number of Allocation IDs must match the number of subnets that are used for the load balancer.
+	// Each entry in the eipAllocations list must be unique.
+	// A maximum of 10 IP allocations are permitted.
```

Comment:
This is an improvement, just one small nit pick. We refer to EIPs in 4 different ways here:
1. eipAllocations (the field `eipAllocations`)
2. EIP (an Elastic IP address generically)
3. Allocation ID (the ID of a EIP that will be allocated)
4. IP Allocation (an EIP that will be allocated)

Most of these are interchangeable, as they all generally refer to the same thing, an EIP Allocation as it pertains to this this field. I think it's best to be consistent to avoid any confusion that we are still generally talking about the same thing.

Here's what I came up with, use `eipAllocations` when you want to refer precisely to this field, `EIP` when you generically want to refer to an EIP, and `EIP allocation` when you are referring to items in this `eipAllocations` field/list. Use your judgement or preference.

```suggestion
	// eipAllocations can only be used with external scope, not internal.
	// An EIP can be allocated to only a single IngressController.
	// The number of EIP allocations must match the number of subnets that are used for the load balancer.
	// Each EIP allocation must be unique.
	// A maximum of 10 EIP allocations are permitted.
```

Reply:
done. I am ok with this suggestion.